src/services/DFGVariable.jl

```diff
@@ -332,6 +334,7 @@ end
 
 Set solver data structure stored in a variable.
 """
+#TODO Repeated functionality?
```

Comment:
this looks like its repeated in add/update

Reply:
if so we should delete the duplicate

Reply:
There are a few cases where functionality is duplicated. I deprecated some and aliased other. From slack Sam voted for deprecate for maintenance and I tend to agree.